composer.json

```diff
@@ -66,13 +67,14 @@
             "src/bootstrap.php"
         ]
     },
-    "minimum-stability": "stable",
     "scripts": {
-        "grumphp": "./vendor/bin/grumphp run"
+        "analyse": "phpstan analyse --level 8 src",
```

Comment:
Hello !

Why moving away from Psalm to PHPStan ?

I know better PHPStan too so it's not a big deal, just want to know why ^^

Also why not using the max level instead of 8 ?

Reply:
> Why moving away from Psalm to PHPStan?

In the past I had conflicting errors from Psalm and PHPStan (they contradicted each other). So I only picked PHPStan.

> Also why not using the max level instead of 8?

Because we would have to fix the following errors. Those can probably be fixed, but I'm not sure if it's worth it.

```
 ------ ----------------------------------------------------------------------------------------------------------------------------- 
  Line   CodeCoverageExtension.php                                                                                                    
 ------ ----------------------------------------------------------------------------------------------------------------------------- 
  111    Argument of an invalid type mixed supplied for foreach, only iterables are supported.                                        
  123    Parameter #1 $lowUpperBound of class SebastianBergmann\CodeCoverage\Report\Text constructor expects int, mixed given.        
  124    Parameter #2 $highLowerBound of class SebastianBergmann\CodeCoverage\Report\Text constructor expects int, mixed given.       
  125    Parameter #3 $showUncoveredFiles of class SebastianBergmann\CodeCoverage\Report\Text constructor expects bool, mixed given.  
  126    Parameter #4 $showOnlySummary of class SebastianBergmann\CodeCoverage\Report\Text constructor expects bool, mixed given.     
 ------ ----------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------- 
  Line   Listener/CodeCoverageListener.php                                                                                                    
 ------ ------------------------------------------------------------------------------------------------------------------------------------- 
  114    Cannot access offset string on mixed.                                                                                                
  114    Cannot call method process() on mixed.                                                                                               
  153    Argument of an invalid type mixed supplied for foreach, only iterables are supported.                                                
  154    Parameter #1 $option of method FriendsOfPhpSpec\PhpSpec\CodeCoverage\Listener\CodeCoverageListener::filterDirectoryParams() expects  
         array<string, string>|string, mixed given.                                                                                           
  159    Argument of an invalid type mixed supplied for foreach, only iterables are supported.                                                
  160    Parameter #1 $option of method FriendsOfPhpSpec\PhpSpec\CodeCoverage\Listener\CodeCoverageListener::filterDirectoryParams() expects  
         array<string, string>|string, mixed given.                                                                                           
  165    Parameter #1 $filenames of method SebastianBergmann\CodeCoverage\Filter::includeFiles() expects array, mixed given.                  
  167    Argument of an invalid type mixed supplied for foreach, only iterables are supported.                                                
  168    Parameter #1 $filename of method SebastianBergmann\CodeCoverage\Filter::excludeFile() expects string, mixed given.                   
 ------ ------------------------------------------------------------------------------------------------------------------------------------- 
```